pipelines/controller/src/test/java/com/google/fhir/analytics/FlinkConfigurationTest.java

```diff
@@ -28,16 +28,16 @@
 import org.apache.flink.configuration.GlobalConfiguration;
 import org.apache.flink.configuration.MemorySize;
 import org.apache.flink.configuration.TaskManagerOptions;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
```

Comment:
Do we know why the coverage issues is only triggered for `controller`? Is it because of Spring?

Reply:
I had a deeper look at this. Looks like the `spring-boot-starter-test` dependency in `controller` module pulls the `Junit 5` jars and the `Junit 5` platform does not run any `Junit 4` test cases by default as per this [documentation](https://maven.apache.org/surefire/maven-surefire-plugin/examples/junit-platform.html). So, to run `Junit 4` test cases as well we need to include this dependency `junit-vintage-engine`. I have made this change in the latest commit.

However, I have retained my changes of migrating to `Junit 5`, so that when we migrate to `Junit 5` these changes need not be done again.